### PR TITLE
Implementation of new colour scheme

### PR DESCRIPTION
**Pull request by BenPoole88**
I've added all the new colours we've decided on for FMCG, so we can see how they look across the site initially, and we can tweak them as we go.

Screenshot 2020-07-13 at 10 27 51 (2)

### CODE REVIEW

File: web/themes/custom/fmcg_material/css/stylec.css [hunk 1]
```diff
@@ -0,0 +1,6004 @@
+@charset "UTF-8";
```
Comment (rlmumford) [inline web/themes/custom/fmcg_material/css/stylec.css]: This looks like a mistype to me (the file name not the line)

**Review by rlmumford — CHANGES_REQUESTED** [review]
There are clearly other changes in here (more than just the colour).

I think we need to think what's yellow, whats blue and what's red. Can you do some work on that? (Maybe the "Dashboard" button is blue, and the logout button is red for example)

File: web/themes/custom/fmcg_material/css/stylec.css (hunk 1 above)
Comment (BenPoole88) [reply web/themes/custom/fmcg_material/css/stylec.css]: Ah yes that's correct!

**Comment by BenPoole88** [comment]
> There are clearly other changes in here (more than just the colour).
> 
> I think we need to think what's yellow, whats blue and what's red. Can you do some work on that? (Maybe the "Dashboard" button is blue, and the logout button is red for example)

Yep that's fine, I'm going to devote some time to that today and provide some examples.

**Comment by BenPoole88** [comment]
So I've done a few examples of the colours in-situ. I've done a mockup of adding the white slash in the navbar too as James mentioned using that, with the different navigation areas in the different brand colours (and also utilising that across the site, e.g. Yellow specifically for Jobs, Teal for About Us etc). There's a couple of different backgrounds too, although I don't think James is fully sold on the icons in general:

White Slash Navbar

Jobs Page 2

Jobs Page

Background 2

Background 1

See what you think!

**Comment by rlmumford** [comment]
I think the background image either needs to be single color, or massively faded out!

**Comment by rlmumford** [comment]
I like the white swoop on the menu bar, yellow doesn't work on white though.

**Comment by BenPoole88** [comment]
> I like the white swoop on the menu bar, yellow doesn't work on white though.

Definitely agree on this!

**Comment by BenPoole88** [comment]
> I think the background image either needs to be single color, or massively faded out!

For sure. I think I'd actually go with single colour and faded out

**Pull request merged**